src/App.tsx

```diff
@@ -1,50 +1,77 @@
-import React from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
-
-export const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
+import { useState } from 'react';
+import { goodsArrayFromServer } from './types/GoodsFromServer';
+import { SortType } from './types/SortType';
 
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType | ''>('');
+  const [isReverse, setIsReverse] = useState(false);
+
+  const visibleGoods = [...goodsArrayFromServer].sort((good1, good2) => {
+    switch (sortField) {
+      case SortType.Alphabetically:
+        return good1.localeCompare(good2);
+      case SortType.Length:
+        return good1.length - good2.length;
+      default:
+        return SortType.Default;
+    }
+  });
+
+  const reset = () => {
+    setSortField('');
+    setIsReverse(false);
+  };
+
+  if (isReverse) {
+    visibleGoods.reverse();
+  }
+
   return (
     <div className="section content">
       <div className="buttons">
-        <button type="button" className="button is-info is-light">
+        <button
+          onClick={() => setSortField(SortType.Alphabetically)}
+          type="button"
+          className={`button is-info ${sortField === 'alphabetically' ? '' : 'is-light'}`}
```

Comment:
The checklist item [UX] recommends marking the active button to indicate to the user what action is active. In the className, you're checking `sortField === 'alphabetically'`, but it should be `sortField === SortType.Alphabetically` to match the enum value.